### PR TITLE
pr_27

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Review by sns-seb2 — CHANGES_REQUESTED** [review]
SonarQube analysis found issues:
Bug Bugs: 2
Vulnerability Vulnerabilities: 0
Code Smell Code Smells: 1

[See all issues in SonarQube](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&branch=pr_27&resolved=false)

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAx0E7PPHgtnqdCFELC###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_27@9
The return value of "length" must be used.

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:8]: ###AWAx0E7PPHgtnqdCFELB###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_27@6
Complete the task associated to this TODO comment.

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAx0E7PPHgtnqdCFELD###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_27@9
A "NullPointerException" could be thrown; "message" is nullable here.

**Comment by sns-seb** [comment]
FOO

[1]: http://github.com 'Foo bar'
[2]: http://github.com 'Foo bar'
[3]: ab 'Foo bar'

**Pull request closed without merging**